sdk/messaging/azwebpubsub/healthapi_client_custom.go

```diff
@@ -0,0 +1,35 @@
+//go:build go1.18
+// +build go1.18
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package azwebpubsub
+
+import (
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/runtime"
+	"github.com/Azure/azure-sdk-for-go/sdk/messaging/azwebpubsub/internal"
+)
+
+// HealthAPIClientOptions contains optional settings for [HealthAPIClient]
+type HealthAPIClientOptions struct {
+	azcore.ClientOptions
+}
+
+// NewHealthAPIClient creates a client that checks the healthy status of Web PubSub service
+func NewHealthAPIClient(endpoint string, options *HealthAPIClientOptions) (*HealthAPIClient, error) {
```

Comment:
This doesn't require any kind of authentication? Will it ever? Assuming that's the case, then I think this should be named `NewHealthAPIClientWithNoCredential` per our [design guidelines](https://azure.github.io/azure-sdk/golang_introduction.html#service-client-constructors).